localization/localization-spanish/README.md

```diff
@@ -0,0 +1,143 @@
+# Tabla de contenidos
+
+* [Guía de estilo (este documento)](README.md)
+* [Glosario de términos](glossary.md)
+* [Prácticas recomendadas](best-practices.md)
+* [Errores comunes de traducción](common-errors.md)
+
+# Guía de estilo de elementary OS en español
+
+Bienvenidas y bienvenidos a la wiki del equipo de traducción de elementary OS al español. Esta guía habla de detalles específicos de elementary para la localización al español. Este documento pretende unificar las traducciones de todas las variantes del español y nos ayuda a tener una traducción coherente y unificada para elementary en nuestro idioma.
+
+# Sobre esta guía
+
+_Esta guía no es una biblia inmutable e indiscutible._
+
+Es un conjunto de referencias y acuerdos entre las personas que han traducido elmentary al español que nos resultan convenientes a la hora de traducir el sistema operativo y sus aplicaciones. Y debido a que el español está en continua evolución, las recomendaciones que consignemos aquí hoy pueden cambiar.
+
+Si tiene una petición, queja, reclamo o sugerencia constructiva, es más que bienvenida. Recomendamos iniciar la discusión en:
```

Comment:
Love this whole section and the choice to start by inviting collaboration 👏🏼

Reply:
Thanks, I think the elementary ethos tends to be on this side always, thus the comment :)